resources/graphql/authorization.graphql

```diff
@@ -38,6 +39,11 @@ type Organization {
   users: [UserInfo]
 }
 
+type Token {
+  token: [String]
```

Comment:
Shouldn't this be called tokens (plural) ?

Reply:
agreed, I have changed this to "Tokens"